.travis.yml

```diff
@@ -2,6 +2,8 @@ sudo: required
 language: python
 services:
   - rabbitmq
+env:
+  - FLASK_SECRET_KEY=H5uMeOWaXO6iuEQxn9DR8GWutA9oZxWQ DATABASE_URL=sqlite:////home/travis/db
```

Comment:
Let's just add a comment here saying this isn't actually a secret.

Reply:
Oh, I figured out why it's required -- similar issue to 0fb9a869a860672a80de832e37c81902bbaf8cb3 with ordering of imports and configuration in `test/runtests.py`.